sources/quest_2/chapter_2/step_10/step_10.md

```diff
@@ -18,5 +18,15 @@ fun test(arg1: u64, arg2: 64) {
 
 Here `arg1` of type `u64` and `arg1` of type `u64` are the two parameters declared as a part of the function.
 
+
+🤔 You might have noticed the variable `house` is written as `_house` . Do you know why?
+
+**Note:** While you try to compile the newly created function you will recieve a warning saying `Unused assignment or binding for local 'house'. Consider removing, replacing with '_', or prefixing with '_' (e.g., '_house')` 
+
+Concept : Using `_` acknowledges that the variable is intentionally not being used. We will talk more about this in the later chapter once we start dealing with local variables.
+
+
 ---
-Let us play around with the function a little more, create a function named `init_house` with parameters to initialise the house struct. We will cover signers in detail shortly.
\ No newline at end of file
+Let us play around with the function a little more, create a function named `init_house` with parameters to initialise the house struct. We will cover signers in detail shortly.
+
```

Comment:
As we saw last time, it's important to give the user as much guidance as possible, especially with function names, arguments, etc...

Here we'd put something like :

Let us play around with the function a little more, create a function named `init_house` with parameters to initialise the house struct. The parameters are named `s` for the signer, as well as `members` and `location`. I'll let you guess their type! This function must initialize a house... with all its parameters!
 
We will cover signers in detail shortly.

Reply:
Keep in mind that the user does not have access to the "target" part during the exercise, but only if he fails to complete the development.

The target area acts as a layer for our verification system, so it's important to guide the user to the right names so that we can validate their work.